lib/timeutils/strptime-tz.c

```diff
@@ -349,6 +351,26 @@ strptime_with_tz(const char *buf, const char *fmt, struct tm *tm, long *tm_gmtof
           state |= S_MDAY;
           continue;
 
+        case 'f':
+        {
+          const unsigned char *end = conv_num(bp, tm_usec, 0, 999999);
+          int digits = end - bp;
+
+          /*
+           * If we read less than 6 digits, we need to adjust the value:
+           * "012" was parsed as 12 but is 12000 us.
+           */
+          while (digits++ < 6)
+            {
+              *tm_usec *= 10;
+            }
+
+          bp = end;
+          LEGAL_ALT(ALT_O);
+          state |= S_USEC;
+        }
+        continue;
+
```

Comment:
`astyle(1)` get very confused about this new code and insists in pretending that it should be indented that way :confused: As a result, the `continue` has the same indentation level as the `case` statements…

Reply:
just add the continue inside the braces.